src/main/webapp/index.html

```diff
@@ -22,6 +22,20 @@
 		    });
         drawTableWithBooks();
 	});
+	
+	$(document).ready(function() {

```

Comment:
There is already the ready handler above this one. Isn't it a better practice to put this there?